python/hail/expr/expression.py

```diff
@@ -3151,7 +3151,7 @@ def alt_alleles(self):
         --------
         .. doctest::
 
-            >>> eval_expr(variant.alt_allele())
+            >>> eval_expr(variant.alt_alleles())
```

Comment:
does this have to be `variant.alt_alleles`?

Reply:
good catch! fixed.